packages/components/src/mobile/global-styles-context/utils.native.js

```diff
@@ -286,6 +309,19 @@ function normalizeFontSizes( fontSizes ) {
 	return normalizedFontSizes;
 }
 
+export function useMobileGlobalStylesColors() {
+	const colorGradientSettings = useMultipleOriginColorsAndGradients();
+	const availableThemeColors = colorGradientSettings.colors
+		.reduce( ( colors, origin ) => colors.concat( origin.colors ), [] )
+		.reverse();
```

Comment:
> With this ordering, Custom colors are placed at the end of the list. I'm wondering if since they're custom, we should put them at the beginning, wdyt?

Yeah, I was wondering about that and ended up putting them at the end since they were at the end in the web editor. But maybe it'd make sense to have them first since users would probably want to use those more.

> In any case, I think that with the introduction of different categories (i.e. Theme, Default and Custom), we should display them by groups as it's done in the web version, instead of a single row:

Great suggestion! I created a [new issue](https://github.com/WordPress/gutenberg/issues/38525) to get some design help for that.